quickrun.el

```diff
@@ -465,7 +465,13 @@ FMT and ARGS passed `message'."
                  (:description . "Compile Kotlin file and execute")))
     ("crystal" . ((:command . "crystal")
                   (:compile-only . "%c build %s")
-                  (:description . "Run Crystal program"))))
+                  (:description . "Run Crystal program")))
+
+    ("v" . ((:command . "v")
+	    (:exec . "%c run %o %s %a")
+	    (:tempfile . nil)
+	    (:remove "%n")
+	    (:description . "Compile and run V programs"))))
```

Comment:
No problem. The rest of the file uses spaces, so I made sure to space the V section with spaces. Is that OK?

Reply:
Yeah, make sure the indentation is correct too! Thanks! 👍

Reply:
I don't see the changes here. You sure you have commit the changes? 😕 Thanks!

Reply:
Hmm. I guess I went through and added spaces and the changes didn't register because nothing changed. I'm still not sure what the indentation style is here. I aligned the V line with the rest of the code. 

The indents are different based on the name of the language, as seen here: https://github.com/emacsorphanage/quickrun/blob/master/quickrun.el#L452-L468.